lib/ui/app_bar.dart

```diff
@@ -54,27 +70,62 @@ abstract class CustomAppBar {
 
   static Widget profileAction() {
     return IconButton(
-        icon: Icon(Icons.account_circle, color: Color(primaryGreen), size: appBarIconSize),
+        icon: Icon(Icons.account_circle,
+            color: Color(primaryGreen), size: appBarIconSize),
         onPressed: () {});
   }
 
+  static Widget shareAction(String articleID) {
```

Comment:
Why we have the share in the open article?

Reply:
the share functionality was merged in with the open article story